### PR TITLE
Features/caritas add address to alternate contact #159338204

### PR DESCRIPTION
**Pull request by fddayan**
https://www.pivotaltracker.com/story/show/159338204

### CODE REVIEW

File: app/javascript/utils/objectUtils.js [hunk 1]
```diff
@@ -1,6 +1,22 @@
-import { forEach, isFunction } from 'lodash'
+import {
+  forEach,
```
Comment (akegan) [inline app/javascript/utils/objectUtils.js:2]: why not just call them all as _.forEach, etc?

**Review by akegan — COMMENTED** [review]
@fddayan looks like the DOB PR caused merge conflicts in one of the files, otherwise looks good. going to test it now.

**Review by akegan — DISMISSED** [review]
Info shows up in salesforce, but not in the application preview after you've saved the application
image
image

File: app/javascript/utils/objectUtils.js (hunk 1 above)
Comment (fddayan) [reply app/javascript/utils/objectUtils.js:2]: if we import individually, webpack only include in the bundle the functions that we use and not the entire lodash library. Our bundle size is smaller. More info here https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark

**Review by akegan — APPROVED** [review]

**Pull request merged**